.gitignore

```diff
@@ -7,7 +10,7 @@ build/
 DerivedData/
 
 ## Documentation
-docs/
+docs/public_documentation
```

Comment:
Just a suggestion, not critical.
```suggestion
docs/output
```

Reply:
Updated output dir's name to `jazzy_output` as you recommend.